gymnasium_robotics/envs/adroit_hand/hammer.py

```diff
@@ -133,6 +135,11 @@ class AdroitHandHammerEnv(MujocoEnv, EzPickle):
 
        reward=lift_hammer+hammer_nail-0.1*get_to_hammer-take_hammer_head_to_nail-
 
+    The `sparse` reward variant of the environment can be initialized by calling `gym.make('AdroitHandHammerSparse-v*')`.
```

Comment:
Preferably lets pin to the latest version v1 since v0 didn't have this feature

Reply:
Fixed